zammad/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: zammad
-version: 15.4.2
+version: 15.4.3
```

New feature should raise minor version


```suggestion
version: 15.5.0
```